src/main/resources/templates/repo/main-repo-shared-resources-template.json.vpt

```diff
@@ -382,7 +382,8 @@
 								},
 								{
 									"Action": [
-										"kms:Decrypt"
+										"kms:Decrypt",
+										"kms:ListAliases"
```

Comment:
Can this be blamed on https://github.com/Sage-Bionetworks/Synapse-Stack-Builder/commit/689041fb3cefb52dff2990c32407cb3baa526787? I removed the kms:Decrypt from the firehose role, note that it didn't have the kms:ListAliases though so I wonder if this is actually still needed for the firehose role (maybe it appears only when firehose tries to push records to S3)?